lib/plugins/simpleRetry.js

```diff
@@ -17,30 +21,70 @@ var simpleRetry = function(worker, func, queue, job, args, options){
     self.queueObject = self.worker;
   }
 
-  self.sleep = 1000;
-  if(self.options.sleep){ self.sleep = self.options.sleep; }
+  if (self.args) {
+    jobHash = crypto.createHash('sha256').update(JSON.stringify(self.args)).digest('hex');
+    self.retryKey = self.queueObject.connection.key('retryTimes', jobHash);
```

Comment:
please stick to the existing key conventions: `reque:*` prefix and all lower-case